frankenphp.c

```diff
@@ -448,6 +448,20 @@ static size_t frankenphp_read_post(char *buffer, size_t count_bytes)
 	return go_read_post(ctx->current_request, buffer, count_bytes);
 }
 
+static void frankenphp_flush(void *server_context)
+{
+	frankenphp_server_context* ctx = SG(server_context);
+
+	sapi_send_headers();
+
+	if(ctx->finished) {
+		// todo: do we have a signal from go that the connection is aborted? If so:
+		// php_handle_aborted_connection();
+	}
```

Comment:
I imagine the whole block will be optimized out during compilation, but we can comment the whole thing out (or remove it).